dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -1565,6 +1730,9 @@ class updatedns
             case 10:
                 $error = "Dynamic DNS ({$this->_dnsHost}): No change in my IP address and/or " . $this->_dnsMaxCacheAgeDays . " days has not passed. Not updating dynamic DNS entry.";
                 break;
+            case 11:
+                $error = "Dynamic DNS: (ERROR!) Mandatory configuration options are missing in update profile.";
```

Comment:
this may be too unspecific for the user

Reply:
I needed a check state for missing / empty "resource id" for the API input. I can make the error specific to the test as "missing resourceid / customer id", no problem. 
But as input validation is mandatory from the WEB GUI, this should only happen when the config.xml is corrupted or manually edited (or code get's screwed up :yum:), so user should never see the errors _missing { **username, password, zoneid, etc.** }_
I thought of this as catch-all for the dyndns providers which are missing information, thus blocking the curl request from even being sent.  (**more** specific than default "_Unknown Response._")

To resolve:
- Route A: make this error exactly specific for missing input field
- Route B: leave as is and add another hint "Re-check/Re-enter your input fields in GUI as maybe something went wrong with saving"
- Route C: use same error case:8 as done with 'azure' plugin

Can we go with A and try to rework the dyndns framework later on? 😀